agent/util-scripts/pbench-tool-meister-start

```diff
@@ -1055,7 +1061,8 @@ def main(_prog: str, cli_params: Namespace) -> int:
             sysinfo_path = benchmark_run_dir / "sysinfo" / "beg"
             try:
                 sysinfo_path.mkdir(parents=True)
-            except Exception:
+            except Exception as e:
+                logger.debug("Unable to create sysinfo-dump directory {}: {}", sysinfo_path, e)
```

Comment:
Ouch; no, we don't, and I've been dinged before on doing it where not necessary. However, I wrote this just because there was another nearby `except` that wrote to the debug log in addition to the stderr/file `error_log` and it seemed to "make sense" to be consistent ... but looking at it now I realize that I used *server* syntax `{}` rather than *agent* syntax `%s` and this won't actually work. 😦

Reply:
Hurray for code review!  (Nevermind that I found the wrong bug, I found a bug!!  🤣)